doc/man7/EVP_SIGNATURE-ML-DSA.pod

```diff
@@ -66,6 +66,12 @@ The default value of 0 causes the per message randomness to be randomly
 generated using a DRBG. Setting this to 1 causes the per message randomness
 to be set to 32 bytes of zeros. This value is ignored if "test-entropy" is set.
 
+=item "mu" (B<OSSL_SIGNATURE_PARAM_MU>) <integer>
+
+The default value of 0 causes sign and verify operations to process a raw message.
+Setting this to 1 causes those operations to assume the input is the mu value
```

Comment:
Cosmetic: 
```suggestion
Setting this to 1 causes those operations to assume the input is the C<mu> value
```
Optionally, could be greek, provided there's a `=encoding utf-8` at the top of the POD file.

Reply:
We don't use encodings anywhere else, so I'll just add the `C<>`.